Signal/src/ViewControllers/ThreadViewHelper.m

```diff
@@ -0,0 +1,136 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "ThreadViewHelper.h"
+#import <SignalServiceKit/TSDatabaseView.h>
+#import <SignalServiceKit/TSStorageManager.h>
+#import <SignalServiceKit/TSThread.h>
+#import <YapDatabase/YapDatabaseConnection.h>
+#import <YapDatabase/YapDatabaseViewChange.h>
+#import <YapDatabase/YapDatabaseViewConnection.h>
```

Comment:
nullability annotations here and in the .h (just in case my previous comment was lost in the rebase)